clang/lib/CIR/CodeGen/CIRGenBuiltin.cpp

```diff
@@ -1580,18 +1580,21 @@ RValue CIRGenFunction::emitBuiltinExpr(const GlobalDecl GD, unsigned BuiltinID,
     llvm_unreachable("BI__builtin_wmemcmp NYI");
   case Builtin::BI__builtin_dwarf_cfa:
     llvm_unreachable("BI__builtin_dwarf_cfa NYI");
-  case Builtin::BI__builtin_return_address: {
+  case Builtin::BI__builtin_return_address:
+  case Builtin::BI__builtin_frame_address: {
     mlir::Location loc = getLoc(E->getExprLoc());
     mlir::Attribute levelAttr = ConstantEmitter(*this).emitAbstract(
         E->getArg(0), E->getArg(0)->getType());
     int64_t level = mlir::cast<cir::IntAttr>(levelAttr).getSInt();
```

Comment:
This isn't your change, but why are we getting a singed int and storing to `int64_t`? The level should be an `unsigned int` and the intrinsic arg is a `UInt32`.